workflowoverview.php

```diff
@@ -176,10 +185,11 @@
     'finishdelay' => format_time($workflow->finishdelay),
     'delayglobally' => $workflow->delayforallworkflows,
     'trigger' => array_values($triggers),
-    'automatic' => $displaytotaltriggered,
-    'coursestriggered' => $amounts['all']->triggered,
-    'coursesexcluded' => $amounts['all']->excluded,
-    'coursesetsize' => $amounts['all']->coursesetsize,
+    'showcoursecounts' => $showcoursecounts,
+    'automatic' => $showcoursecounts ?? $displaytotaltriggered,
+    'coursestriggered' => $showcoursecounts ?? $amounts['all']->triggered,
+    'coursesexcluded' => $showcoursecounts ?? $amounts['all']->excluded,
+    'coursesetsize' => $showcoursecounts ?? $amounts['all']->coursesetsize,
```

Comment:
`$a ?? $b` returns `$a` only if `isset($a) && $a !== null` and `$b` otherwise. So even if showcoursecounts is enabled, 'automatic' and the 'course*' fields will always just be set to `'1'`.

Depending on your preferences, you can use for example `... => $amounts['all']->triggered ?? null` (since ?? luckily tests for `isset`, there will be no warnings that way), or `... => $showcoursecounts ? $amounts['all']->triggered : null`, or just move those values out of the initial array creation and set them seperately only `if ($showcoursecounts) {...}`.

Reply:
Oh wow, that was definitely a brain fart, sorry for that and thank you for spotting :+1:

I of course tested only one course in a workflow and - guess what - it "worked" :see_no_evil: :smile:

Reply:
No worries, thats what reviews are for :D